README.md

```diff
@@ -1 +1,10 @@
 # kerbero
```

Comment:
usually we write the how to start the project in the readme

Reply:
Yup, @salvanicola add a section like
```markdown
# How to run the project
....
```